tools/build_defs/hash/sha256_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Copyright 2016 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+cat > input.txt <<EOF
+01234567890123456789012345678
+EOF
+
+cp input.txt tmp.txt
+for i in {1..22}; do
```

Comment:
Please add a comment about why the loop goes to 22 in particular.

Reply:
Thanks for the review.

I just did it to have a 120MB file.  The 120MB is arbitrary, but enough to be meaningful.

Reply:
Thanks. Can you please extract the test file creation logic to a function? I think that'd improve readability.